resources/public/api/conf/1.0/application.yaml

```
@@ -39,5 +39,7 @@ paths:
     $ref: "./checkpoints_create.yaml"
   /individuals/self-assesment-test-support/vendor-state/checkpoints/{checkpointId}:
     $ref: "./checkpoints_delete.yaml"
+  /individuals/self-assesment-test-support/vendor-state/checkpoints:
```

Should we add the nino query parameter?
```suggestion
  /individuals/self-assesment-test-support/vendor-state/checkpoints?nino={nino}:
```